python/rmmbr/main_test.py

```diff
@@ -7,6 +7,14 @@
 from rmmbr import cloud_cache, local_cache, mem_cache
 
 
+async def wait_all_tasks():
+    all_tasks = asyncio.all_tasks()
+    if current_task := asyncio.current_task():
+        all_tasks.remove(current_task)
+    if all_tasks:
```

Comment:
iiuc this `if` is redundant, because waiting on an empty set will just return

Reply:
what i expected too but it throws an exception instead =\